docs: improve contributing section (#980)
ansys/pyansys-geometry: doc/source/getting_started/existing/index.rst

.. warning::

   Running a SpaceClaim or Discovery normal session does not suffice to be
   able to use it with PyAnsys Geometry. Both products need the ApiServer extension to be

>    running. In this case, users are recommended to launch the products directly from the PyAnsys

Reviewer: ```suggestion
   running. In this case, to ease the process, you should launch the products directly from the PyAnsys
```

>    Geometry library as shown in :ref:`ref_creating_local_session` to ease the process.

Reviewer: ```suggestion
   Geometry library as shown in :ref:`ref_creating_local_session`.
```
